src/components/attendee/AttendeeHeader/AttendeeHeader.tsx

```diff
@@ -18,14 +18,23 @@ import { NavOverlay } from "../NavOverlay/NavOverlay";
 
```

Comment:
Imports should be absolute
` from "components/`

Better still, some kind of organization scheme that doesn't make us put one component inside the directory of another. Maybe `./src/components/nav/` as a parent for all nav related components

Reply:
fixed [2ebc1f3](https://github.com/sparkletown/sparkle/pull/2827/commits/2ebc1f31eb8b1121e844036f240cd927eeb6d714)